File: v2/01/teht1.js
/*
function toCelsius(fahrenheit) {
    return (5/9) * (fahrenheit-32);
}
function area(radius)
{
    return Math.PI * radius * radius;
};
 */

let toCelsius = fahrenheit => (5 / 9) * (fahrenheit - 32);

let area = radius => Math.PI * radius ** 2;

console.log(toCelsius(100));
console.log(area(100));

File: v3/01/JavaApp/src/teht1.java
public class teht1 {

    public static void main(String args[]) {
        MyOperation toCelsius = fahrenheit -> (fahrenheit - 32) * 5 / 9;
        MyOperation area = radius -> Math.PI * radius * radius;

        System.out.println(toCelsius.operation(100));
        System.out.println(area.operation(10));


    }

    public interface MyOperation {
        public double operation(double x);
    }

}

